src/main/java/ExitCommand.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * Represents an exit command.
+ */
+public class ExitCommand extends Command {
+
+    /**
+     * Executes the exit command to say goodbye to the user and save the tasks.
+     * @param tasks list of tasks
+     * @param ui user interface to display deleted message
+     * @param storage file storage
+     */
+    public void execute(TaskList tasks, Ui ui, Storage storage) {
+        storage.saveTasks(tasks.getTasksList());
```

Comment:
Perhaps you could call `saveTasks` after every time the task list is modified instead of just when the user exits the programme. This would ensure that the tasks are saved even when the program is closed incorrectly, e.g. by pressing the close button instead of typing "bye". However, this is only a suggestion for your consideration.